onnxruntime/test/onnx/main.cc

```diff
@@ -314,7 +314,12 @@ int real_main(int argc, char* argv[], OrtEnv** p_env) {
       {"tf_inception_v4", "Cast opset 9 not supported yet"},
       {"tf_nasnet_large", "disable temporarily"},
       {"tf_nasnet_mobile", "disable temporarily"},
-      {"tf_pnasnet_large", "disable temporarily"}};
+      {"tf_pnasnet_large", "disable temporarily"},
+      {"dynamic_slice_neg", "op deprecated in version 10"},
```

Comment:
Even the op is deprecated, we should still be able to run the old models, right?

Reply:
Unit tests are still there... this is to disable onnx_test_runner tests....
Right now onnx_test_runner is failing with this error: Op registered for DynamicSlice is depracted in domain_version of 10
What is the right way to get this working? Should I create a versioned kernel for DynamicSlice registered from opset 1 to 9? may be this way version 10 wont be picked up?

One more thing... these tests are deleted from onnx because this op is removed from onnx. How do we sync the test data in onnxruntime? after the sync these tests wont run anyways...

Reply:
We need to test our code against **every** ONNX release, to make sure we still can run the old models. Therefore, you shouldn't filter out any test unless you will fix the test soon.